keyboards/melgeek/mach80/keymaps/default/keymap.c

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2020 MelGeek <melgeek001365@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[0] = LAYOUT_tkl_ansi( /* Base */
+        KC_GESC,            KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,   KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_PSCR,  KC_SLCK,
+        KC_GRV,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,    KC_7,     KC_8,     KC_9,     KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,  KC_INS,   KC_HOME,
+        KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,    KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,  KC_BSLS,  KC_DEL,   KC_END,
+        KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,    KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,  KC_ENT,   KC_PAUS,  KC_PGUP,  KC_PGDN,
+        KC_LSFT,  KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,    KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RSFT,                      KC_UP,
+        KC_LCTL,  KC_LGUI,  KC_LALT,                      KC_SPC,                                MO(1),    KC_RALT,	 KC_RGUI,  KC_RCTL,  KC_LEFT,  KC_DOWN,  KC_RGHT
+	),
```

Comment:
This does not look like the standard `tkl_ansi` layout — in the standard layout the nav cluster has 3 columns, and this layout somehow has mostly 2 columns and an extra row there (and the Pause key stuffed in an unusual location in that extra row). It also does not match what Google finds for “Mach80” (which looks like a completely standard TKL layout).

Normally any weird matrix wiring should be concealed inside the layout macro, and the order of the layout macro arguments should mostly resemble the physical locations of the keys. The order of the layout macro arguments must also match the order of key entries in `info.json`, otherwise the configurator won't be able to make the keymap properly. And if you are using a standard layout name, such as `tkl_ansi`, the order of keys must also match that standard layout.